errors.txt

```diff
@@ -0,0 +1,548 @@
+rec0BmROCIFxyURjs failed mapping/validation. Error message: No mapped location for Barcelona,+Catalonia,+Spain. Record fields: {"post_website":"https://www.barcelona.cat/covid19/en/financial-support?p=creation-cecore","post_language":["English","Catalan","Spanish"],"status":"Live in Progress","Pandemia":"COVID-19","post_type":["Information","Funding"],"post_name":"Creation of the Coordination Centre for the Economic Response (CECORE)","share_with":"City","post_description":"The Coordination Centre for the Economic Response (CECORE) has been set up to activate Barcelona’s economy in response to the situation generated by Covid-19.\n\nThe purpose of the CECORE is to monitor the economic situation in the city to improve decision-making, keep track of the situation with the municipal budget, develop action to help reactivate the economy, coordinate municipal action with social and economic stakeholders and forge alliances with the rest of administrations.\n\nThe centre will implement specific activation measures in each economic sphere: business, enterprise and the self-employed; tourism and creative industries; markets, commerce and consumption; social and solidarity economy; city promotion and branding; quality employment; industrial reactivation and sports and economic activity. \n\n \n\nCampaign to foster local consumption and improve the promotion of Barcelona\n\nA major local, national and international communication campaign is also being started to back local consumption and relaunch Barcelona’s international profile.\n\nOn a local and national level the goal is to boost local consumption in the short to mid term. On an international level, the intention is to promote Barcelona abroad as soon as the circumstances allow.\n\n \n\nProposed socio-economic agreement for Barcelona\n\nThe municipal government has also reached out to municipal opposition groups to construct a major socio-economic city agreement with the following aims:\n\nWork to reactivate the economy.\nAgree on action and measures.\nOvercome the crisis as soon as possible.\nMove forward in the transformation to a greener and digitally-based model.\nFurther informationOpen in a new window\n\nLatest update 17/04/2020","post_objective":["Offer"],"post_tags":["Economics"],"post_duration":"Forever","country":["Spain"],"state":["Catalonia"],"city":["Barcelona"],"from_whom":["Goverment"]}.

```

Comment:
Do we need this file?

Reply:
I think it is just from your local? so let's remove it

Reply:
Oopsie. Def not. I will rewrite commit so it will be gone from history.

Reply:
Sounds good :) Yeah in this cases comes in handy removing that commit from history.

Reply:
Oops used the wrong command that completely changed older commit shas. I should have rebased instead I guess.